### PR TITLE
Update and rename open-template.md to open-proposal-energymeter.md

### PR DESCRIPTION
**Pull request by saulwiggin**

### CODE REVIEW

File: oep-proposals/open-proposal-energymeter.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
+To submit a proposal, please create a PR against this template in this repo. Please title your file `open-proposal-title.md`, replacing `title` with the name of your project.
```
Comment (raullenchai) [inline oep-proposals/open-proposal-energymeter.md:1]: Thanks for the application, definitely looking interesting! Can you pls create "open-proposal-energymeter.md" while keeping renaming "open-template.md"?

File: oep-proposals/open-proposal-energymeter.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
+To submit a proposal, please create a PR against this template in this repo. Please title your file `open-proposal-title.md`, replacing `title` with the name of your project.
+
+# Open Grant Proposal: `Energy Meter`
+
+**Name of Project:** Using IoTeX Cryptocurrency to
```
Comment (raullenchai) [inline oep-proposals/open-proposal-energymeter.md:5]: Name incomplete

File: oep-proposals/open-proposal-energymeter.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+
+IoTeX branded energy metereing would add a new product line to your existing products such as pebble. This would provide additional income and provide a global, crypto solution to the digital energy market. It could be possible to incorperate the hardware and web solutions with payment into additional functionality for crypto and branding. 
+
+## Deliverables
+
+We would be proposing a new product branded with your cryptocurrency. This would include the hardware and the software (web) as well as payment systems and inclusion into the cryptocurrency infrastructure. We could discuss a commision percentage for money processed through the system. 
+
+## Development Roadmap
+
+The hardware is largly developed and software. It would require branding and perhaps a package design with yourselves. The cryptocurrency inclusion would need to be developed which could be done by myself and take a few months.
```
Comment (raullenchai) [inline oep-proposals/open-proposal-energymeter.md:29]: Interesting! Do you have a link/pic/spec of the hardware?

Comment (raullenchai) [reply oep-proposals/open-proposal-energymeter.md:29]: In addition, pls detail the roadmap here - how exactly IOTX or IoTeX will be used to support this meter? Thanks.

**Comment by saulwiggin** [comment]
Hi, any news on this proposal?

**Comment by raullenchai** [comment]
Thanks for the proposal! The team has reviewed this proposal on 6/5 and here is the feedback:

We like the idea to make IOTX part of an IoT devices and the details of this integration is not clear in the proposal. We would suggest adding the following details to make it complete:
- how IOTX will be used with this energy meter? Is it to trade energy in a P2P fashion in a smart grid?
- what is the tech stack of this hardware and how to integrate with IoTeX tech stack within

If more clarification is needed, we would be happy to zoom with you - http://raullen.youcanbook.me/. Thanks!

From core-dev

**Comment by saulwiggin** [comment]
Hi Raullen,

I would be happy to do a zoom chat for discuss further.

I was thinking of integrating it into a payment system we were doing for
the energy meters which would be written in Javascript. In addition it
could connect to IOTX and the payment could be processed in the
cryptocurrency with the user having to buy coins to pay for the energy.
This could be then generalised into a marketplace where energy could be
bought and sold in a smart grid.

I have booked a time for this.

Kind Regards,
Saul

**Comment by raullenchai** [comment]
Assume application withdrew.

**Pull request closed without merging**